charts/kyverno-policies/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: kyverno-policies
-version: v0.0.0
+version: v0.0.1
```

Comment:
I don't think we should change this file. It should be changed when we cut a new version.